eppo_client/variation_type.py

```diff
@@ -16,7 +16,7 @@ def is_expected_type(
         if expected_variation_type == cls.STRING:
             return isinstance(assigned_variation.typedValue, str)
         elif expected_variation_type == cls.NUMERIC:
-            return isinstance(assigned_variation.typedValue, Number)
+            return isinstance(assigned_variation.typedValue, Number) and not isinstance(assigned_variation.typedValue, bool)
```

Comment:
I used `Number` so I wouldn't have to list out the number types (`int`, `float`, etc), but looks like this also included booleans.

```
>>> from numbers import Number
>>> isinstance(False, Number)
True
>>> isinstance(True, Number)
True
>>> isinstance('asdf', Number)
False
>>> isinstance({'foo': 'bar'}, Number)
False
```